packages/s/Sendbird.json

```
@@ -0,0 +1,27 @@
+  
+{
+  "name": "sendbird",
```

```suggestion
  "name": "Sendbird",
```
`name` should match the filename. Either `name` needs to become `Sendbird` or the fle should be renamed to `sendbird.json`.